backend/ccm/canvas_api/enroll_users.py

```diff
@@ -50,6 +51,9 @@ def enroll_user(canvasapi: Canvas, section_id: int, login_id: str, role: str):
             f"sections/{section_id}/enrollments",
             _kwargs=list(enrollment_params.items())
         )
-        return Enrollment(section._requester, response.json())
+        allowed_fields = ['id', 'course_id', 'course_section_id', 'user_id', 'type']
+        enrollment_result = Enrollment(section._requester, response.json())
+        serializer = CanvasObjectROSerializer(enrollment_result, allowed_fields=allowed_fields)
+        return serializer.data
```

Comment:
Did not realize that not returning the serializer response for enroll user, so changed to fit the existing pattern